test/JDBC/expected/ISC-Tables-vu-verify.out

```diff
@@ -12,6 +12,15 @@ master#!#dbo#!#isc_tables_vu_prepare_t1#!#BASE TABLE
 ~~END~~
 
 
+-- Table types should not be a result
+-- Should not return any rows.
+SELECT * FROM information_schema.tables WHERE TABLE_NAME = 'isc_table_type1'
```

Comment:
Lets add similar test for information_schema.views also.

Reply:
table types are not showing up in information_schema.views. Added test cases to test  sysdatabases should not be a part of isc.views.